Probesys/bileto
# imp: Various visual improvements

## Related issue(s)



N/A

## Changes



See commits

## How to test manually



- navigate in the app
- check everything looks fine

## Checklist
